src/utils/hooks/useQRCodeScan.tsx

```diff
@@ -18,34 +18,35 @@ export function useQRCodeScan(
 
         useEffect(() => {
             async function start() {
-                if (permission !== 'granted') return
+                if (permission !== 'granted' || !video.current) return
                 try {
-                    const device = await getBackVideoDeviceId()
-                    const media = await navigator.mediaDevices.getUserMedia({
-                        audio: false,
-                        video: device === null ? { facingMode: 'environment' } : { deviceId: device },
-                    })
-                    setMediaStream(media)
-                    if (!video.current) return
+                    let media = mediaStream
+                    if (!media) {
+                        const device = await getBackVideoDeviceId()
+                        media = await navigator.mediaDevices.getUserMedia({
+                            audio: false,
+                            video: device === null ? { facingMode: 'environment' } : { deviceId: device },
+                        })
+                        setMediaStream(media)
+                    }
                     video.current.srcObject = media
```

Comment:
AbortError: The fetching process for the media resource was aborted by the user agent at the user's request.

Reply:
It is ok to have this message logged.

Reply:
And..... It's not working.

Reply:
Maybe you did not grant your camera's permission.

Reply:
Camera permission is granted, and got this error while I clicked 'start scanning' in the storybook